overview.md

```diff
@@ -8,24 +8,18 @@ gcam-version: v5.1
+GCAM is an integrated, multi-sector model that explores both human and Earth system dynamics. The role of models like GCAM is to bring multiple human and physical Earth systems together in one place to shed light on system interactions and provide scientific insights that would not otherwise be available from the pursuit of traditional disciplinary scientific research alone. GCAM is constructed to explore these interactions in a single computational platform with a sufficiently low computational requirement to allow for broad explorations of scenarios and uncertainties. Components of GCAM are designed to capture the behavior of human and physical systems, but they do not necessarily include the most detailed process-scale representations of its constituent components. On the other hand,  model components in principle provide a faithful representation of the best current scientific understanding of underlying behavior.
 
-<img src="gcam-figs/overview_fig1.png" width="600"><br/>
-Figure 1: Illustrative Schematic of IA modeling and research, linking human and Earth systems. Note that not all IA models represent all of the systems shown in this illustrative figure.
-{: .fig}
-
-In general IAMs can be divided into two categories, those that are highly aggregated and whose core problem is the conduct of global, century-scale, cost-benefit analysis (examples include DICE, FUND, and PAGE), and higher resolution models whose core problem is understanding the details of human and physical Earth system interactions. GCAM is a member of the latter class of more highly-resolved IA models.
-
-IA models like GCAM do not predict the future. They instead provide "what if" scenarios, quantifying the implications of different possible future conditions. GCAM reads in external "scenario assumptions" about key drivers such as population, economic activity, technology, and policies. GCAM can then assess the implications of these assumptions on, for example, commodity prices, energy use, land use, water use, emissions, and concentrations. 
+No model of systems as complex as the ones that GCAM explores can predict the future, and GCAM is no exception. Models like GCAM are intended to provide "what if" scenarios, quantifying the implications of different possible future conditions. GCAM reads in external "scenario assumptions" about key drivers (e.g., population, economic activity, technology, and policies) and then assesses the implications of these assumptions on key scientific or decision-relevant outcomes (e.g., commodity prices, energy use, land use, water use, emissions, and concentrations). 
```

Comment:
I really don't like the first sentence in this paragraph.  I get what we are trying to say here, but to me this sounds like the kind of thing people say when they're trying to deflect criticism.  Tonally, I think we want to go for matter-of-fact, while this comes across as more defensive.  Maybe we could replace the first two sentences with something like:
> GCAM allows users to explore what-if scenarios, quantifying the implications of possible future conditions.  These outputs are not predictions of the future; they are a way of analyzing the potential impacts of different assumptions about future conditions.
In other words, we should go for something that explains what the model is trying to do without making it sound like forecasting is the default purpose of modeling, and any departure from that needs defending.